ArduPlane/Attitude.cpp

```diff
@@ -388,7 +388,9 @@ void Plane::stabilize()
           during transition to vtol in a tailsitter try to raise the
           nose rapidly while keeping the wings level
          */
-        nav_pitch_cd = constrain_float((quadplane.tailsitter.transition_angle+5)*100, 5500, 8500),
+        float transition_rate = quadplane.get_tailsitter_transition_angle_vtol()/ float(quadplane.tailsitter.transition_time_vtol_ms/2);
```

Comment:
I wonder if it would be more intuitive to have the param being the rate directly. We could then calculate the timeout internally.

Reply:
guess you could say the same for the other transition also?

Reply:
yeah, I think they should both be rates

Reply:
well we can have forward and back have angle and time for now, and param convert the angle to a rate based on the old angle and time, if we change both in the future,right?

Reply:
Yeah, we can change in the future